### PR TITLE
[WIP] Remove integration with SUSEFirewall in preparation for working with …

### PR DESCRIPTION
**Pull request by HouzuoGuo**
…firewalld. Bump version to 3.2.

### CODE REVIEW

**Comment by teclator** [comment]
@HouzuoGuo probably you do not have time to also work on this PR, and once @mchf increased the version of the package in #7 the concficts needs to be also solved

File: package/yast2-vpn.changes [hunk 1]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Nov  9 15:39:33 UTC 2017 - hguo@suse.com
+
+- Remove integration with SUSEFirewall in preparation for working
+  with firewalld.
```
Comment (teclator) [inline package/yast2-vpn.changes:5]: A bug number or Fate ID is missing.

**Comment by HouzuoGuo** [comment]
greetings from Helsinki.

I'm busy like a bee over here, please get in touch with Peter Varkoly or another colleague if this one needs urgent attention.

**Comment by teclator** [comment]
@varkoly could you also take care of this PR?

**Comment by teclator** [comment]
@varkoly I will close the PR by now as it has conflicts and has not received attentions by now feel free to reopen it or just create a new one for the adaptations needed for firewalld

**Pull request closed without merging**